### ISSUE TITLE
 Issue with cluster-level resources

### ISSUE THREAD

**Issue by rcohenDBRS** [justinbarrick/fluxcloud#26]
I've been seeing this issue a lot lately:
```
ts=2019-03-19T18:15:52.144930335Z caller=loop.go:412 component=sync-loop err="executing HTTP request: 400 Bad Request parsing <cluster>:clusterrole/kube2iam: invalid service ID\n"
```
It causes `fluxctl sync` to return `Error: timeout`, but no other discernible issues.

See https://github.com/weaveworks/flux/issues/1843

**Comment by kraney** [comment]
It also prevents the flux-sync tag from being moved forward.

**Comment by kraney** [comment]
This error is actually coming from flux code, and is already fixed in flux.

**Comment by justinbarrick** [comment]
I will get to this eventually, but as this is an open source project anyone can bump the version.

**Comment by kraney** [comment]
Yes that was meant as a tip to people they can resolve it that way (I have) not as a complaint. Thanks, fluxcloud does a nice job

**Comment by rocktavious** [comment]
Seeing this with justinbarrick/fluxcloud:v0.3.4 && quay.io/weaveworks/flux:1.11.1 and quay.io/weaveworks/flux:1.12

```
flux-665cffc4fb-bfkjx fluxcloud {"id":0,"serviceIDs":["\u003ccluster\u003e:namespace/infra-integ","\u003ccluster\u003e:namespace/infra-prod"],"type":"sync","startedAt":"2019-04-11T22:05:58.564240083Z","endedAt":"2019-04-11T22:05:58.564240083Z","logLevel":"info","metadata":{"commits":[{"revision":"f956e3c6a3ee444b8ff36dad1dae60fc68245d14","message":"test"},{"revision":"4f691aa2e00f8d46d419edd907bfa6853b89d220","message":"add the gke cluster folder"},{"revision":"9a5b243217ecd69bee8616064105e41688cacc56","message":"Add global folder structure for apps, and warning about clusters folder (#7)"}],"includes":{"other":true},"initialSync":true}}
flux-665cffc4fb-bfkjx fluxcloud parsing <cluster>:namespace/infra-integ: invalid service ID
flux-665cffc4fb-bfkjx flux ts=2019-04-11T22:05:59.575092664Z caller=loop.go:412 component=sync-loop err="executing HTTP request: 400 Bad Request parsing <cluster>:namespace/infra-integ: invalid service ID\n"
flux-665cffc4fb-bfkjx flux ts=2019-04-11T22:05:59.588819181Z caller=loop.go:90 component=sync-loop err="executing HTTP request: 400 Bad Request parsing <cluster>:namespace/infra-integ: invalid service ID\n"
```

**Comment by davidkarlsen** [comment]
@justinbarrick which version needs bumping?

**Comment by davidkarlsen** [comment]
Seems like this was fixed with v0.3.6

**Comment by rocktavious** [comment]
Can confirm fixed in v0.3.6 with flux 1.11.1
